### DISCUSSION TITLE
Settings - cannot change own password

### DISCUSSION THREAD

**Discussion by mbiesiad** [plausible/analytics#3102] [Anything Else]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Plausible Cloud from plausible.io

### Describe the bug

Hi,
At this moment user cannot change password in easy way.
Repro steps:
1. As logged in user go to Settings https://plausible.io/settings
2. See no possibility to Change password there

Additional information: workaround is to use Reset password feature, but it takes more time and it's just workaround.

Best wishes,

### Expected behavior

User can easy change own password from Settings tab.

### Screenshots

NA

### Environment

```markdown
NA
```

**Comment by mbiesiad** [comment]
"[metmarkosaric](https://github.com/metmarkosaric) converted this issue into discussion [#3102 ](https://github.com/plausible/analytics/discussions/3102)[43 minutes ago](https://github.com/plausible/analytics/issues/3101#event-9700162437)"

it is type of Security issue of course. 

Best wishes,

**Reply by metmarkosaric** [reply]
thanks @mbiesiad! true! i've moved it here as it's the better place in terms of how we plan the development. we keep the issues list to the actual bugs with the product only and the forum here for planning any improvements or new feature development. thank you for sharing this. it's something we plan to improve in the future.

**Comment by ErlandssonA** [comment]
Would be very nice to see this being implemented in both the cloud version as well as the self-hosted version.

I know it is planned, but do we have any updates/progress on it?

**Reply by redbmk** [reply]
yeah is this marked as resolved because it was implemented in the cloud version? I'm not seeing it as an option in the self-hosted version.

**Reply by ruslandoga** [reply]
👋 @redbmk 

According to the changelog it was implemented in v2.1.4. What version are you using?

**Reply by redbmk** [reply]
v2.1.3 😂 🙈 

Awesome, just updated to v2.1.4 and the user settings page looks completely different now 💯
